src/vs/base/browser/ui/breadcrumbs/breadcrumbsWidget.ts

```diff
@@ -3,17 +3,21 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import 'vs/css!./breadcrumbsWidget';
 import * as dom from 'vs/base/browser/dom';
+import { DataTransfers } from 'vs/base/browser/dnd';
 import { IMouseEvent } from 'vs/base/browser/mouseEvent';
 import { DomScrollableElement } from 'vs/base/browser/ui/scrollbar/scrollableElement';
 import { commonPrefixLength } from 'vs/base/common/arrays';
-import { ThemeIcon } from 'vs/base/common/themables';
 import { Emitter, Event } from 'vs/base/common/event';
-import { DisposableStore, dispose, IDisposable } from 'vs/base/common/lifecycle';
+import { DisposableStore, IDisposable, dispose } from 'vs/base/common/lifecycle';
 import { ScrollbarVisibility } from 'vs/base/common/scrollable';
-import 'vs/css!./breadcrumbsWidget';
+import { ThemeIcon } from 'vs/base/common/themables';
+import { FileKind } from 'vs/platform/files/common/files';
+import { FileElement, OutlineElement2 } from 'vs/workbench/browser/parts/editor/breadcrumbsModel';
```

Comment:
This introduces layer breakers. Code in `/base` cannot access code from `/workbench` or `/platform`. See https://github.com/microsoft/vscode/wiki/Source-Code-Organization#layers for more details

Reply:
thanks！I will correct here